### PR TITLE
Numerous improvements, particularly Windows path handling

### PR DESCRIPTION
**Pull request by bbb999**
Numerous improvements, particularly Windows path handling

### CODE REVIEW

File: PProPanel/jsx/Premiere.jsx [hunk 1]
```diff
@@ -392,88 +410,97 @@ $._ext_PPRO={
...
+			var projPath   	= new File(app.project.path);
+			var outputPath  = Folder.selectDialog("Choose the output directory");
+
+			if (outputPath != null && projPath.exists){
 			
-			if (qe.platform == 'Macintosh') {
-				var out_preset_path = "/Applications/Adobe Premiere Pro CC 2014/Adobe Premiere Pro CC 2014.app/MediaIO/systempresets/58444341_4d584658/XDCAMHD 50 NTSC 60i.epr";
-				sep = '/';
+				if (Folder.fs == 'Macintosh') {
+					var outputPresetPath = "/Applications/Adobe\ Premiere\ Pro\ CC\ 2015/Adobe\ Premiere\ Pro\ CC\ 2015.app/Contents/MediaIO/systempresets/58444341_4d584658/XDCAMHD\ 50\ NTSC\ 60i.epr";
```
Comment (lesavage) [inline PProPanel/jsx/Premiere.jsx]: Should you be mentioning 2015 prior to the release?

File: PProPanel/index.html [hunk 2]
```diff
@@ -22,15 +21,26 @@
...
+            var path    	= csInterface.getSystemPath(SystemPath.EXTENSION);
+
+			if (OSVersion != null){
+				if (OSVersion.indexOf("Windows") >=0){
+					var sep = '\\\\';
+					path = path.replace(/\//g, sep);
+            } else {
+              var sep = '/';
+            }
+           		path = path + sep + 'payloads' + sep + 'PProPanel.sqpreset';
```
Comment (lesavage) [inline PProPanel/index.html]: you could write:
path += sep + 'payloads' + sep + 'PProPanel.sqpreset';

Comment (bbb999) [inline PProPanel/index.html]: Fixed! I'd read something scary about += operators in JavaScript ages back, but that seems to work fine.

File: PProPanel/jsx/Premiere.jsx (hunk 1 above)
Comment (bbb999) [inline PProPanel/jsx/Premiere.jsx]: I don't think I've given away too much. If any rumor sites pop up, stating that they know the name and number of the next release of PPro—it's all on me. :)

**Pull request merged**